### PR TITLE
RELATED: BB-721 Fix space between value and underline for headline

### PR DESCRIPTION
**Pull request by martinmosko**

### CODE REVIEW

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/653/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/428/ : SUCCESS in 55s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/151/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/654/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/309/ : UNSTABLE in 5m 48s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/655/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/429/ : SUCCESS in 59s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/656/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/430/ : SUCCESS in 50s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/657/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/431/ : SUCCESS in 54s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/658/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/432/ : SUCCESS in 57s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/659/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/433/ : SUCCESS in 49s

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/152/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/660/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/310/ : UNSTABLE in 5m 07s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/661/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/434/ : SUCCESS in 58s

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/153/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/662/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/311/ : SUCCESS in 4m 03s

File: stories/Headline.jsx [hunk 1]
```diff
@@ -38,14 +38,33 @@ storiesOf('Headline components/Headline', module)
                         primaryItem: {
                             uri: 'myId',
                             value: '666429.405',
-                            format: '[color=9c46b5][backgroundColor=d2ccde]$#,##0.00',
+                            format: '[color=9c46b5][backgroundColor=d2ccde]$#,##0.00 happy group',
```
Comment (petrjanu) [inline stories/Headline.jsx]: happy group?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/663/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/435/ : SUCCESS in 54s

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/154/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/664/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/312/ : UNSTABLE in 4m 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/665/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/436/ : SUCCESS in 1m 01s

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/155/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/666/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/313/ : SUCCESS in 4m 05s

**Comment by petrjanu** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/667/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/438/ : SUCCESS in 57s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-merge-compliance-check/119/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/439/ : SUCCESS in 1m 04s

**Pull request merged**